src/VisualStudio/IntegrationTest/IntegrationTestBuildProject.csproj

```diff
@@ -1,8 +1,13 @@
 <Project Sdk="Microsoft.Build.Traversal">
   <PropertyGroup>
-   <TargetFrameworks>net472</TargetFrameworks>
+    <TargetFrameworks>net472</TargetFrameworks>
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net45" />
```

Comment:
Why do we need the net45 reference assemblies and not e.g. the net472 ones?

Reply:
Not sure, but this fixes the issue.